### PR TITLE
Test the handlers

### PR DESCRIPTION
**Pull request by annasambrook**
Adding tests for handlers.

### CODE REVIEW

**Review by 0xArdi — APPROVED** [review]
Changes look good. Some CI issues.

File: packages/valory/skills/check_stop_trading_abci/skill.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ description: This skill implements the check for stop trading for an AEA.
 license: Apache-2.0
 aea_version: '>=1.0.0, <2.0.0'
 fingerprint:
+  .coverage: bafybeiavhhydxlqau64cx2n56z7orprble2olsy5zxycnr7qrw2s4vrbge
```
Comment (Adamantios) [inline packages/valory/skills/check_stop_trading_abci/skill.yaml:9]: This should not be present.

File: packages/valory/skills/decision_maker_abci/skill.yaml [hunk 2]
```diff
@@ -53,12 +53,14 @@ fingerprint:
   states/sampling.py: bafybeihriyyskmlupjiwytx2pdpftms7plkjku5bip64c3ztx7oi3n43ci
   states/tool_selection.py: bafybeiak5ihuie4nxh3sguiea6pcdgyxr4k4xyzvq6o2uj5xpf7urocawy
   tests/__init__.py: bafybeiakpi3k3kc7wrjj7hrluvjcj36lu2gezpmrctwiz5yg2fe7ggnf3i
+  tests/behaviours/.coverage: bafybeigpapif6g2wseb6rgh53m7bxti4lnnfyhmepudcyx2c7gxgurqxai
```
Comment (Adamantios) [inline packages/valory/skills/decision_maker_abci/skill.yaml:56]: Same here.

File: packages/valory/skills/market_manager_abci/skill.yaml [hunk 3]
```diff
@@ -6,6 +6,7 @@ description: This skill implements the MarketManager for an AEA.
 license: Apache-2.0
 aea_version: '>=1.0.0, <2.0.0'
 fingerprint:
+  .coverage: bafybeihh6nwmpbwn5qk6r2t3wsbbmxpyoyj2taai3tiabbvjde26v5ryg4
```
Comment (Adamantios) [inline packages/valory/skills/market_manager_abci/skill.yaml:9]: Same.

File: packages/valory/skills/tx_settlement_multiplexer_abci/skill.yaml [hunk 4]
```diff
@@ -6,6 +6,7 @@ description: This skill implements a multiplexer for the transaction settlement
 license: Apache-2.0
 aea_version: '>=1.0.0, <2.0.0'
 fingerprint:
+  .coverage: bafybeicx45ygnllbjejkfo3xp5hvtz3ztohtcdxy2ki7mq3gcypvvo6i2u
```
Comment (Adamantios) [inline packages/valory/skills/tx_settlement_multiplexer_abci/skill.yaml:9]: Same

File: packages/valory/skills/check_stop_trading_abci/tests/test_handlers.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    ],
+)
+def test_handler(handler: Handler, base_handler: Handler) -> None:
+    """Test that the 'handlers.py' of the CheckStopTradingAbci can be imported."""
+    handler = handler(
+        name="dummy_handler",
+        skill_context=MagicMock(skill_id=PublicId.from_str("dummy/skill:0.1.0")),
+    )
+
+    assert isinstance(handler, base_handler)
```
Comment (Adamantios) [inline packages/valory/skills/check_stop_trading_abci/tests/test_handlers.py:76]:
```suggestion
    handler_ = handler(
        name="dummy_handler",
        skill_context=MagicMock(skill_id=PublicId.from_str("dummy/skill:0.1.0")),
    )

    assert isinstance(handler_, base_handler)
```

Comment (annasambrook) [reply packages/valory/skills/check_stop_trading_abci/tests/test_handlers.py:76]: mypy is not complaining so not making this change

**Review by Adamantios — APPROVED** [review]

**Pull request merged**